### ISSUE TITLE
Add Moderator/Administrator/Patreon

### ISSUE THREAD

**Issue by botea14** [adikanchukov/los-santos-v#13]
Hi, Sorry for being a "noob", but i want to ask how can i give Mod / Admin permissions and add Patreon Supporters 1/2/3 ? I'm using this mode for a few days and i really like it! Thanks.

**Comment by botea14** [comment]
Also got another issue. When i was in a Wanted mission, a friend joined, we killed each other for a few times, but all kills were counted as suicide for the one who was dead.

When we got closer to each other, game crashed for both with following message.

[Sun May  3 03:16:27 2020] [Session] [Info] Dropped { VESPIUTA, 3, Game crashed: mockingbird-potato-september (GTA5+16038A5) }
[Sun May  3 03:16:27 2020] [Session] [Info] Dropped { aLe TRADEIT.GG, 2, Game crashed: mockingbird-potato-september (GTA5+16038A5) }
